app/usedgoodreads/jobs.py

```diff
@@ -112,13 +112,12 @@ def single_searchresult_to_bookoffer(item, isbn):
     details = item.find_element_by_class_name("aditem-details")
     # date = item.find_element_by_class_name("aditem-addon").text
     price = details.find_element_by_tag_name("strong")
-    price_int = price.text.split()
-    price = 10**9  # TODO: :)
+    price_float = price.text.replace(".", "").split()
```

Comment:
Why the split? Was there a case with >1 element?

Reply:
The split was there before and below we are iterating over all items; you've implemented that, not sure why it's this way :P